pyne/_dagmc.pyx

```diff
@@ -9,7 +9,7 @@ from pyne.utils import QA_warn
 cimport numpy as np
 import numpy as np
 
-from pyne cimport cpp_dagmc_bridge
+cimport cpp_dagmc_bridge
```

Comment:
I'm not sure I understand why this change and/or how it works...

Reply:
scikit-build-core uses cmake to build cython modules, and I found that the build fails if we don't switch to this. It is unable to find cpp_dagmc_bridge.